web/app/fr/gospeak/web/GospeakApplicationLoader.scala

```diff
@@ -83,7 +83,7 @@ class GospeakComponents(context: ApplicationLoader.Context)
   lazy val messageBus: MessageBus[GospeakMessage] = wire[BasicMessageBus[GospeakMessage]]
   lazy val orgaMessageBus: GospeakMessageBus = wire[GospeakMessageBus]
   lazy val messageHandler: MessageHandler = wire[MessageHandler]
-
+  lazy val contactRepo: ContactRepo = db.contact
```

Comment:
can be better to have it next to the other repos

Reply:
Done